artifactory/cli.go

```diff
@@ -885,17 +883,42 @@ func getDockerFlags() []cli.Flag {
 	flags = append(flags, getSkipLoginFlag())
 	return flags
 }
+func getcommondepracatedFlages() []cli.Flag {
+	return []cli.Flag{
 
-//These flags are not valid for native npm command
-func getNpmLegacyFlages() []cli.Flag {
-	npmFlags := []cli.Flag{
 		cli.StringFlag{
-			Name:  "npm-args",
-			Usage: "[Optional] A list of npm arguments and options in the form of \"--arg1=value1 --arg2=value2\"` `",
+			Name:  "user",
+			Usage: "[depracated][Optional] Artifactory username.` `",
+		},
+		cli.StringFlag{
+			Name:  "password",
+			Usage: "[depracated][Optional] Artifactory password.` `",
+		},
+		cli.StringFlag{
+			Name:  "apikey",
+			Usage: "[depracated][Optional] Artifactory API key.` `",
+		},
+		cli.StringFlag{
+			Name:  "access-token",
+			Usage: "[depracated][Optional] Artifactory access token.` `",
+		},
+		cli.BoolFlag{
+			Name:  "insecure-tls",
+			Usage: "[depracated][Default: false] Set to true to skip TLS certificates verification.` `",
+		},
+		cli.StringFlag{
+			Name:  "url",
```

Comment:
It probably makes more sense to have the url as the first flag in this function.

Reply:
Ok.